src/main/java/frc/robot/RobotContainer.java

```diff
@@ -56,12 +57,12 @@ public class RobotContainer {
    * ************
    */
 
-  public static DriveRobotOriented driveRobotOriented = new DriveRobotOriented();
-  public static DriveFieldOriented driveFieldOriented = new DriveFieldOriented();
-  public static EnterXMode enterXMode = new EnterXMode();
-  public static DeployPneumatics deployPneumatics = new DeployPneumatics();
-  public static RunBelts runBelts = new RunBelts();
-  public static RunIntake runIntake = new RunIntake();
+  private static DriveRobotOriented driveRobotOriented = new DriveRobotOriented();
```

Comment:
Make these not static, having them static is gonna super break stuff. I'm surprised this even compiled.